rotkehlchen/db/dbhandler.py

```diff
@@ -2073,11 +2074,16 @@ def get_entries_count(
         cursor.execute(cursorstr)
         return cursor.fetchone()[0]
 
-    def delete_data_for_ethereum_address(self, write_cursor: 'DBCursor', address: ChecksumEvmAddress) -> None:  # noqa: E501
-        """Deletes all ethereum related data from the DB for a single ethereum address"""
-        other_eth_accounts = self.get_blockchain_accounts(write_cursor).eth
-        if address in other_eth_accounts:
-            other_eth_accounts.remove(address)
+    def delete_data_for_evm_address(
+            self,
+            write_cursor: 'DBCursor',
+            address: ChecksumEvmAddress,
+            blockchain: SUPPORTED_EVM_CHAINS,
+    ) -> None:
+        """Deletes all evm related data from the DB for a single evm address"""
+        other_chain_accounts = self.get_blockchain_accounts(write_cursor).get(blockchain)
+        if address in other_chain_accounts:  # pylint: disable=unsupported-membership-test
+            other_chain_accounts.remove(address)
```

Comment:
I know these 3 lines were here before ... but what do they do?

Before they returned an `other_eth_accounts` struct that's created only in `self.get_blockchain_accounts`. And then if the address is there it's modified.

But ... where is it used? It's not used anywhere in the function.

Unless I read this wrong it is probably a remnant from something else in the past and can go away.